src/lib/utils/offlineUtils.js

```diff
@@ -72,3 +72,10 @@ export const addToOfflineCustomer = async (customer, fieldToAppend, objectToAdd)
         }
     }
 }
+
+// TODO: Return with robust ID generation. 
+// We could probably have all offlineIDs be generate through 1 function if we pass
+// the redux slice / entity to generate the ID for?
+export const generateOfflineId = () => {
+    return (Math.floor(Math.random() * 1000)).toString();
+}
```

Comment:
I was thinking we could jsut make a generic `generateOfflineId` instead of 1 for each object type (Inventory, Customer, CustomerPurchase, etc...) since they'd presumnably be generated via the same routine. I'm not sure what the logic would look like (they might just be case-by-case logic so it might be kinda ugly). lmk what you think or if you prefer to separate them out, I'm good with either approach.

cc @wangannie

Reply:
I was thinking we use generated offline IDs regardless and just use timestamps to match. We'll probably find some problems with whatever method we choose during implementation and we can discuss then?